.github/workflows/final.yml

```diff
@@ -1,24 +1,12 @@
 on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
-  release:
```

Comment:
How does the release fit into this new refactoring?

Reply:
Mistake on my part, I had removed the "on release" trigger but left the release part of the workflow in the file. I re-added the trigger